easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.0.0-foss-2020b.eb

```
@@ -0,0 +1,174 @@
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+builddependencies = [
+    # CMake/oxygen/SWIG are required for building OpenMM
+    ('CMake', '3.18.4'),
+    ('Doxygen', '1.8.20'),
+    # required for installing dm-tree
+    ('Bazel', '3.7.2'),
+    ('SWIG', '4.0.2'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('PyYAML', '5.3.1'),
+    ('TensorFlow', '2.4.1'),
+    ('Biopython', '1.78'),
+    ('HH-suite', '3.3.0'),
+    ('HMMER', '3.3.2'),
+    ('Kalign', '3.3.1'),
+    ('PDBFixer', '1.7'),
+    ('jax', '0.2.19'),  # also provides absl-py
+]
+
+local_openmm_preinstallopts = "export OPENMM_INCLUDE_PATH=%(installdir)s/include && "
+local_openmm_preinstallopts += " export OPENMM_LIB_PATH=%(installdir)s/lib && "
+
+# required to install OpenMM Python API;
+# avoid that setup.py partially uninstalls existing OpenMM Python bindings...
+local_openmm_installopts = " && cd python && sed -i 's/uninstall()/pass/g' setup.py && "
+local_openmm_installopts += "python setup.py build && python setup.py install --prefix=%(installdir)s"
+
+# commit to use for downloading stereo_chemical_props.txt and copy to alphafold/common,
+# see docker/Dockerfile in AlphaFold repository
+local_scp_commit = '7102c6'
+
+components = [
+    # for simtk
+    ('OpenMM', '7.5.1', {
+        'easyblock': 'CMakeMake',
+        'source_urls': ['https://github.com/openmm/openmm/archive/'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}],
+        'patches': [('OpenMM-%(version)s_AlphaFold.patch', 'wrappers/python')],
+        'checksums': [
+            'c88d6946468a2bde2619acb834f57b859b5e114a93093cf562165612e10f4ff7',  # OpenMM-7.5.1.tar.gz
+            '1b109dfff3af5c6aa70690bca14618612953c68840a7e64f679db7ca33c1aff6',  # OpenMM-7.5.1_AlphaFold.patch
+        ],
+        'start_dir': 'openmm-%(version)s',
+        'preinstallopts': local_openmm_preinstallopts,
+        'installopts': local_openmm_installopts,
+    }),
+    (name, version, {
+        'easyblock': 'PythonPackage',
+        'source_urls': [
+            'https://github.com/deepmind/alphafold/archive/refs/tags/',
+            'https://git.scicore.unibas.ch/schwede/openstructure/-/raw/%s/modules/mol/alg/src/' % local_scp_commit,
+        ],
+        'sources': [
+            {
+                'download_filename': 'v%(version)s.tar.gz',
+                'filename': SOURCE_TAR_GZ,
+            },
+            {
+                'download_filename': 'stereo_chemical_props.txt',
+                'filename': 'stereo_chemical_props-%s.txt' % local_scp_commit,
+                'extract_cmd': "cp -a %s .",
+            },
+        ],
+        'patches': [
+            'AlphaFold-%(version)s_fix-packages.patch',
+            'AlphaFold-%(version)s_data-dep-paths.patch',
+            'AlphaFold-%(version)s_n-cpu.patch',
+            'AlphaFold-2.0.0_fix-scp-path.patch',
```

```suggestion
            'AlphaFold-%(version)s_fix-scp-path.patch',
```

Minor consistency nitpick.